README.md

```diff
@@ -58,7 +58,7 @@ To get started with using Wagtail, run the following in a virtual environment:
 
 ![Installing Wagtail](.github/install-animation.gif)
 
-```sh
+```bash
```

Comment:
Please be careful to avoid unrelated changes in other sections.

Also this usage of `sh` is intentional - see https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html#when-using-console-terminal-code-blocks

Reply:
It is ready for another review